documentation/src/main/asciidoc/ch09.asciidoc

```diff
@@ -326,6 +326,35 @@ For instance, this might be useful if you want to replace the default message of
 constraint with a more explicit one.
 ====
 
+When dealing with distributed architectures, you might need some tolerance for the clock skew in the temporal constraints
+e.g. `@Past` or `@Future`.
+
+You can set a clock skew tolerance by bootstrapping your `ValidatorFactory` as below:
```

Comment:
"Clock skew" is an established thing with a different meaning: https://en.wikipedia.org/wiki/Clock_skew

Maybe we should find a different term, e.g. "temporal validation lenience/tolerance" or so?

Reply:
Maybe "temporal tolerance" would be enough?

Reply:
Or maybe `Clock drift`? But both terms refer to the differences in frequency between clocks while the feature is actually about coping with imperfectly synchronized clocks (regardless the reason for the asynchrony itself).